code/modules/mob/living/carbon/human/species.dm

```diff
@@ -894,6 +894,10 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		if(!source.dna.features["ears"] || source.dna.features["ears"] == "None" || source.head && (source.head.flags_inv & HIDEHAIR) || (source.wear_mask && (source.wear_mask.flags_inv & HIDEHAIR)) || !noggin || noggin.status == BODYPART_ROBOTIC)
 			bodyparts_to_add -= "ears"
 
+	if(mutant_bodyparts["pod_hair"])
+		if(!source.dna.features["pod_hair"] || source.dna.features["pod_hair"] == "None" || source.head && (source.head.flags_inv & HIDEHAIR) || (source.wear_mask && (source.wear_mask.flags_inv & HIDEHAIR)) || !noggin || noggin.status == BODYPART_ROBOTIC)
+			bodyparts_to_add -= "pod_hair"
+
```

Comment:
This is unnecessary. I should explain since I am the cause of the confusion.
There's two systems for rendering hair/features: the hardrendering you see here in species.dm, and the external organ system. The external organ systeem is preferable since it's fully modular and not (as) bad. You've coded it for both systems, so you're probably rendering it twice. 

Could you remove the code you wrote for species.dm and see if it still works? If it doesn't just leave it and I'll clean it up later since the whole species rendering system here is on my kill-list

Reply:
Also the reason there are two systems is because the species.dm rendering is being removed and replaced by the external_organs system, but it's a lot of work and I am mostly busy with school and work so you hit the awkward time period where two systems exist for the same thing